basis_set_exchange/tests/common_testvars.py

```diff
@@ -28,6 +28,9 @@
     for ver in v['versions'].keys():
         bs_names_vers.append((k, ver))
 
+# Directory the CLI executables
+cli_dir = os.path.join(_my_dir, '../cli')
```

Comment:
probably should be

```
_parent_dir = os.path.abspath(os.path.join(_my_dir, os.pardir))
cli_dir = os.path.join(_parent_dir, 'cli')
```

or some variation thereof

Reply:
Fixed